docs/semantics.md

```diff
@@ -25,13 +25,21 @@ until they are written out to Cloud Storage, you
 must ensure that there is enough free space available to handle staged content
 when writing large files.
 
-- **Note:** Prior to version 1.2.0, you will notice that an empty file is
-  created in the Cloud Storage bucket as a hold. Upon closing or fsyncing the
-  file, the file is written to your Cloud Storage bucket, with the existing
-  empty file now reflecting the accurate file size and content. Starting with
-  version 1.2, the default behavior is to not create this zero-byte file, which
-  increases write performance. If needed, it can be re-enabled by setting
-  the `create-empty-file: true` configuration in the config file.
+#### Notes
+
+-   Prior to version 1.2.0, you will notice that an empty file is created in the
+    Cloud Storage bucket as a hold. Upon closing or fsyncing the file, the file
+    is written to your Cloud Storage bucket, with the existing empty file now
+    reflecting the accurate file size and content. Starting with version 1.2,
+    the default behavior is to not create this zero-byte file, which increases
+    write performance. If needed, it can be re-enabled by setting the
+    `create-empty-file: true` configuration in the config file.
+-   If the application never sends fsync for a file, it will leave behind its
+    temp-file (in temp-dir), which will not be cleared until the user unmounts
+    the bucket. As an example, if you are writing a large file, and temp-dir
+    does not have enough free space available, then you will get 'out of space'
+    error. Then the temp-file will not be deleted until you do an fsync for that
```

Comment:
@sethiay @mustvicky since we wrote this together, I had second thoughts about this line. the `the temp-file will not be deleted` seems like a break in flow. IMHO this should read. `Then the output of space error will not go away until ...` . 

PTAL

Reply:
What you have currently LGTM. I am fine with the change of replacing the last line with `Then the output of space error will not go away until` but since the primary subject of this whole point is temp-file, it is better to keep it this way IMO.